support/Environments/caltech_hpc_gcc.sh

```diff
@@ -176,6 +176,31 @@ EOF
     fi
     cd $dep_dir
 
+    if [ -f $dep_dir/scotch/lib/libscotch.a ]; then
+        echo "Scotch is already installed"
+    else
+        echo "Installing Scotch..."
+        wget https://gitlab.inria.fr/scotch/scotch/-/archive/v6.1.0/scotch-v6.1.0.tar.bz2
+        tar xjf scotch-v6.1.0.tar.bz2
+        mv scotch-v6.1.0 scotch
+        cd $dep_dir/scotch
+        cp src/Make.inc/Makefile.inc.x86-64_pc_linux2 src/Makefile.inc
+        cd src
+        make -j4
+        cd $dep_dir
+        rm scotch-v6.1.0.tar.bz2
+        echo "Installed Scotch into $dep_dir/scotch"
+        cat >$dep_dir/modules/scotch <<EOF
+#%Module1.0
+prepend-path LIBRARY_PATH "$dep_dir/scotch/lib"
+prepend-path LD_LIBRARY_PATH "$dep_dir/scotch/lib"
+prepend-path CPATH "$dep_dir/scotch/include"
+prepend-path CMAKE_PREFIX_PATH "$dep_dir/scotch/"
+EOF
+    fi
+
+    module use $dep_dir/modules
+    module load scotch
```

Comment:
should there be a `module unload scotch` down by line 285 where the system modules are unloaded?